ui/components/app/signature-request-siwe/index.scss

```diff
@@ -0,0 +1,80 @@
+@import 'signature-request-siwe-header/index';
+@import 'signature-request-siwe-message/index';
+
+.signature-request-siwe {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  min-width: 0;
+  background-color: var(--color-background-default);
+  margin-right: auto;
+  margin-left: auto;
+
+  @media screen and (min-width: $break-large) {
+    width: 408px;
+    max-height: 82vh;
+    min-height: 570px;
+    flex: 0 0 auto;
+    border-radius: 8px;
+    box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+  }
+
+  &__domain-mismatch-warning {
+    padding: 4px 16px;
+  }
+}
+
+
+.signature-request-siwe__domain-mismatch-warning {
+  .no-margin-top {
+    margin-top: 0;
+  }
+
+  .actionable-message svg {
+    width: 13px;
+    height: 13px;
+    left: 10px;
+    top: 20px;
+  }
+}
+
+
+.signature-request-siwe__page-container-footer.page-container__footer {
+  border-top: none;
+}
+
+.signature-request-siwe__warning-popover {
+  .page-container__footer {
+    border-top: none;
+    padding: 0;
+    display: block;
+  }
+
+  .popover-footer {
+    padding: inherit;
+    display: block;
+  }
+
+  &__checkbox-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px 16px 24px;
+
+    &__label {
+      @include H7;
+
+      color: var(--color-text-default);
+      margin-left: 8px;
```

Comment:
`margin-inline-start` may be better for RTL support.

Reply:
Updated